apps/stix-shifter/README.md

```diff
@@ -0,0 +1,84 @@
+STIX-Shifter Threat Bus
+=======================
+
+This app bridges the gap between Threat Bus and commercial security tools by
+leveraging
+[STIX-Shifter](https://github.com/opencybersecurityalliance/stix-shifter).
+
+STIX-Shifter is a tool and library to transform STIX patterns into native
+queries for a variety of (mostly) commercial tools, like
+[IBM QRadar](https://www.ibm.com/security/security-intelligence/qradar) or
+[Splunk](https://www.splunk.com/). Since Threat Bus itself primarily focuses on
+open-source tools, this app now provides a simple way to use intelligence from
+Threat Bus with the commercial tools of your choice.
```

Comment:
```suggestion
[Splunk](https://www.splunk.com/). This app provides a simple way to communicate STIX
Threat Bus with the commercial tools of your choice.
```

As user, I probably run a mix of commercial and open-source tools, so this distinction does not feel relevant.